src/bidiMapper/domains/network/NetworkProcessor.ts

```diff
@@ -367,6 +382,8 @@ export class NetworkProcessor {
           }
 
           try {
+            // TODO: change this as if we have a single param form Pattern
```

Comment:
nit: it's not clear to me what the AI is here. Could you rephrase?

Reply:
It was for me, as I was planning to do more things in the parsing.